amr-wind/wind_energy/actuator/wing/fixed_wing_ops.H

```diff
@@ -36,16 +36,47 @@ struct ReadInputsOp<FixedWing, ActSrcLine>
         pp.get("num_points", wdata.num_pts);
         pp.get("start", wdata.start);
         pp.get("end", wdata.end);
-        pp.query("epsilon", wdata.eps_inp);
-        pp.query("epsilon_chord", wdata.epsilon_chord);
-        pp.get("pitch", wdata.pitch);
+        pp.query_either("epsilon", wdata.eps_inp);
+        pp.query_either("epsilon_chord", wdata.epsilon_chord);
+        if (!pp.contains("pitch_timetable")) {
+            // Constant pitch is required without timetable
+            pp.get("pitch", wdata.pitch);
+        }
         pp.get("airfoil_table", wdata.airfoil_file);
         pp.query("airfoil_type", wdata.airfoil_type);
         pp.queryarr("span_locs", wdata.span_locs);
         pp.queryarr("chord", wdata.chord_inp);
         bool use_fllc = false;
         pp.query("fllc", use_fllc);
 
+        amrex::Vector<int> fcf, dcf;
```

Comment:
I don't immediately see where fcf and dcf are being used. If they are being used, let's move this to where they are being used. If not, let's remove.

Reply:
good catch